contents/product-engineers/llm-product-metrics.md

```diff
@@ -0,0 +1,195 @@
+
+1. Costs are closely tied to app usage. 
+2. Request latency spikes and errors are more common.
+3. LLM output is unpredictable and can lead to a negative user experience.
+
+This means that in addition to [regular product metrics](https://posthog.com/product-engineers/product-health-metrics), you need to monitor LLM-specific ones too. To understand which ones to track, we've put together a list of useful metrics.
+
+The metrics are grouped into three categories: [cost](#cost-related-metrics), [usage](#usage-metrics), and [debugging](#debug-metrics).
+
+> **💡 PostHog tip**: Use our [Langfuse](/docs/llm-analytics/langfuse-posthog) and [Helicone](/docs/llm-analytics/helicone-posthog) integrations to bring your LLM data into PostHog. Alternatively, we have tutorials on how to capture LLM events from [OpenAI](/tutorials/chatgpt-analytics), [Anthropic](/tutorials/anthropic-analytics), and [Cohere](/tutorials/cohere-analytics).
+
+## Cost-related metrics
+
+### Average cost per user
+
+**What is it?** Your total LLM costs divided by the number of active users.
+
+**Why it's useful:** Gives you an idea of how your costs will grow with your product. You can also compare this to revenue per user to understand if your profit margin is viable.
+
+**Questions to ask:**
+- Are average costs going up or down over time? 
+- If they're increasing over time, is it because of increased product usage or increased generation costs?
+- What specific features or types of interactions contribute the most to the cost per user?
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1714744519/posthog.com/contents/average-cost-light.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/v1714744520/posthog.com/contents/average-cost-dark.png"
+  alt="Line chart showing average LLM cost per user over time"
+/>
+
+### P95 cost per user
+
+**What is it?** The top 5% of users who are consuming the most LLM costs.
+
+**Why it's useful:** Determines if specific users disproportionately affect overall costs. You can then investigate whether these costs are justified by their usage, or if there are inefficiencies to address.
+
+**Questions to ask:**
+- What common characteristics or usage patterns do the top 5% of users have?
+- Are there specific feature interactions that result in higher costs for these users?
+- Is the high cost associated with these users sustainable, or does it suggest a need for pricing adjustments?
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1714744211/posthog.com/contents/Screenshot_2024-05-03_at_2.49.39_PM.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/v1714744214/posthog.com/contents/Screenshot_2024-05-03_at_2.50.01_PM.png"
+  alt="Table showing P95 of users who consume the most LLM costs"
+/>
+
+### Average cost per interaction
+
+**What is it?** The cost associated with each request in your LLM features.
```

Comment:
It is slightly unclear if this is cost for using the feature on top of LLMs or directly making the requests to LLMs? I'm not sure this matters, but maybe requires a slight reword.